care_o_bot_desktop/package.xml

```diff
@@ -20,7 +20,7 @@
   <exec_depend>language-pack-en</exec_depend>
   <exec_depend>language-pack-de</exec_depend>
   <exec_depend>openssh-server</exec_depend>
-  <exec_depend>chromium-browser</exec_depend>
+  <!--exec_depend>chromium-browser</exec_depend-->
```

Comment:
no idear? can't you skip fedora in the release process? are all other cob_* packages successfully released for fedora?